### DISCUSSION TITLE
Typo in "Configure the appsettings file"

### DISCUSSION THREAD

**Discussion by artgoldberg** [uwrit/leaf#529] [General]
Hi All

[Configure the appsettings file](https://leafdocs.rit.uw.edu/installation/installation_steps/6_appsettings/) uses the term `SAML2.RoleMapping` 3 times, but the code and its example [`appsettings.json`](https://github.com/uwrit/leaf/blob/master/src/server/API/appsettings.json#L51) uses the term `RolesMapping`.

Regards
Arthur

**Comment by ndobb** [comment]
Thanks Arthur, this has now been corrected.
